balancer-js/src/modules/joins/graph.ts

```diff
@@ -94,11 +94,11 @@ export class PoolGraph {
         // ignore any phantomBpt tokens
         if (pool.tokens[i].address === pool.address) continue;
         const proportion = BigNumber.from(pool.tokens[i].balance)
-          .mul('1000000000000000000')
+          .mul((1e18).toString())
```

Comment:
I find it a bit odd to see this many zeros 😅 , so this is a suggestion to make it more readable. We could also use `WeiPerEther`. What do you think?

Reply:
Yeah, agreed. We'll add a constant for this later I think as I've seen issues with using 1e18 but happy to go with that for now.